ci/install-spellcheck.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+## install-spellcheck.sh: run during a CI build to install Pandoc spellcheck dependencies.
+
+# Set options for extra caution & debugging
+set -o errexit \
+    -o pipefail
+
+sudo apt-get update -y
+sudo apt-get install -y aspell aspell-en
+wget https://raw.githubusercontent.com/pandoc/lua-filters/13c3fa7e97206413609a48a82575cb43137e037f/spellcheck/spellcheck.lua
```

Comment:
I wonder if we want to download `spellcheck.lua` to `build/assets` or `build/pandoc-filters`? Perhaps we should also gitignore whatever file this ends up at in case someone runs this locally.

Reply:
This is also a good idea.  Do you know where pandoc searches for filter files?  I haven't been able to find that documented, and it fails to find `build/assets/spellcheck.lua`.

Reply:
from https://pandoc.org/MANUAL.html

In order of preference, pandoc will look for Lua filters in

1.  a specified full or relative path (executable or non-executable)
2.  `$DATADIR/filters` (executable or non-executable) where `$DATADIR` is the user data directory (see [`--data-dir`](https://pandoc.org/MANUAL.html#option--data-dir), above).

Makes me think we should create a `build/assets/pandoc` directory with a `defaults` and `filter` subdirectory. I can look into this in a new PR. For now you could specify the complete relative path like `build/assets/spellcheck.lua`?

Reply:
> I can look into this in a new PR. 

done in https://github.com/manubot/rootstock/pull/340, but I think these PRs can be merged separately

Reply:
Thanks for starting the pandoc file refactoring.  I'll review it later.

Now that I added the complete relative path, the tests pass again.  This should be ready to merge if you agree.